libraries/engage/locations/components/StoreList/StoreListProduct.jsx

```diff
@@ -1,17 +1,17 @@
+// @flow
 import React, { useContext } from 'react';
 import { ProductImage } from '../../../product';
-import { FulfillmentContext } from '../../locations.context';
+import { type FulfillmentContextProps, FulfillmentContext } from '../../locations.context';
 import StoreListProductName from './StoreListProductName';
-import StoreListProductVariants from './StoreListProductVariants';
-import * as styles from './style';
+import StoreListProductInfo from './StoreListProductInfo';
+import * as styles from './StoreListProduct.style';
 
 /**
  * Renders the product information of the store list.
  * @returns {JSX}
  */
 function StoreListProduct() {
-  const { product } = useContext(FulfillmentContext);
-
+  const { product } : FulfillmentContextProps = useContext(FulfillmentContext);
```

Comment:
You don't need to specify the type of the object here. React's `useContext` will automatically recognise it.

Reply:
Its not for react, its for human code assistance 
![image](https://user-images.githubusercontent.com/18292310/76612881-6cadae00-651d-11ea-8b38-18c127cbb378.png)